Makefile

```diff
@@ -979,6 +979,10 @@ proxy:
 update-deps:
 	$(GO) mod tidy
 
+build-csharp:
+	docker build -t csharp $(REPOSITORY_ROOT)/csharp/OpenMatch
+	(cd $(REPOSITORY_ROOT)/csharp/OpenMatch; docker cp $(shell docker create csharp):/app/. ./.)
```

Comment:
This is somewhat unusual. I think we should build locally. The rest of the makefile builds locally so you have easy access to artifacts.